app/services/books_service.rb

```diff
@@ -0,0 +1,20 @@
+class BooksService
+  def volume(id)
+    get_url("/books/v1/volumes/#{id}")
+  end
+
+  def volumes(keyword)
+    get_url("/books/v1/volumes?q=#{keyword}")
```

Comment:
I'm SO excited about being able to do multiple search types with just a keyword!!